src/busy-indicator.scss

```diff
@@ -0,0 +1,105 @@
+
+@import "./settings";
+@import "./mixins";
+@import "./functions";
+
+$block: #{$fd-namespace}-busy-indicator;
+$fd-dots-color: var(--sapContent_IconColor);
+$fd-dots-color-contrast: var(--sapContent_ContrastIconColor);
+// default sizes
+$fd-dots-size: 0.125rem;
+$fd-dots-margin: 0 0.2rem;
+
+// medium sizes
+$fd-dots-size-m: 0.5rem;
+$fd-dots-margin-m: 0 0.4rem;
+
+// large sizes
+$fd-dots-size-l: 1rem;
+$fd-dots-margin-l: 0 0.75rem;
+
+.#{$block},
+.#{$block}--m,
+.#{$block}--l {
+  @include fd-reset();
+
+  position: relative;
+  display: block;
+  font-size: 0;
+  color: $fd-dots-color;
+}
+
+.#{$block}.contrast,
+.#{$block}--m.contrast,
+.#{$block}--l.contrast {
+  color: $fd-dots-color-contrast;
+
+  .#{$block}--circle-0,
+  .#{$block}--circle-1,
+  .#{$block}--circle-2 {
+    color: $fd-dots-color-contrast;
+  }
+}
+
+.#{$block}--circle-0,
+.#{$block}--circle-1,
+.#{$block}--circle-2 {
+  @include fd-reset();
+
+  color: $fd-dots-color;
+  position: relative;
+  width: $fd-dots-size;
+  height: $fd-dots-size;
+  margin: $fd-dots-margin;
+  display: inline-block;
+  border-radius: 50%;
+  background-color: currentColor;
+}
+
+.#{$block}--circle-0 {
```

Comment:
Is there a way to do this with all sharing the same class name?

Reply:
@jbadan  the animation delay is different and it is good to align